nextcord/enums.py

```diff
@@ -737,6 +738,11 @@ class ForumLayoutType(IntEnum):
     gallery = 2
 
 
+class OnboardingPromptType(IntEnum):
+    MULTIPLE_CHOICE = 0
```

Comment:
I usually prefer capitalised enums, but lowercase would be consistent. Don't forget to document this in api.rst too.